kv_store/src/bank/kv/transaction.rs

```diff
@@ -53,7 +53,6 @@ impl<DB: Database> TransactionKVBank<DB> {
     pub fn upgrade_cache(&mut self) {
         let (set_values, delete) = self.tx.take();
         for (key, value) in set_values {
-            self.block.delete.remove(&key);
```

Comment:
Is that part of the same fix?

Reply:
This is code what should have been removed long ago. It duplicates logic from cache